lib/foreman_salt/plugin.rb

```diff
@@ -24,7 +24,15 @@
        :parent => :configure_menu
 
 
-  # Permissions
+  # Existing permissions
+  p = Foreman::AccessControl.permission(:edit_hostgroups)
+  p.actions << 'foreman_salt/api/v2/salt_hostgroups/#{update}'
+  p.actions << 'hostgroups/#{salt_environment_selected}'
+
+  p = Foreman::AccessControl.permission(:view_hostgroups)
+  p.actions << 'foreman_salt/api/v2/salt_hostgroups/#{show}'
```

Comment:
Why are you putting the actions in `#{}`?

Reply:
You are right, thanks!